src/plugins/ui_stateful/selection.ts

```diff
@@ -552,7 +552,7 @@ export class GridSelectionPlugin extends UIPlugin {
 
     const sheetId = this.getActiveSheetId();
     const handler = new CellClipboardHandler(this.getters, this.dispatch);
-    const data = handler.copy(getClipboardDataPositions(sheetId, target));
+    const data = handler.copy(getFilteredClipboardDataPositions(sheetId, target, this.getters));
```

Comment:
I think this change is useless since it a cut operation here. Which means `getFilteredClipboardDataPositions` is only used once and it can be inlined